src/crimson/os/seastore/transaction.h

```diff
@@ -141,8 +141,24 @@ class Transaction {
     return retired_set;
   }
 
+  enum class src_t {
+    SEASTORE,
+    SEASTORE_READ,
```

Comment:
at first glance, i'd say `SEASTORE` is a superset of `SEASTORE_READ`. if that's the case, why not s/SEASTORE/SEASTORE_WRITE/ or s/SEASTORE/SEASTORE_MUTATE/?

Reply:
renamed to `SEASTORE_MUTATE`